### PR TITLE
Labels highlighted in the text

### LINKED ISSUE #7: Labels not highlighted
The labels in the description are not highlighted

### PR DESCRIPTION
**Pull request by agniutkarsh**
closes #7

### CODE REVIEW

**Review by nitish-awasthi — APPROVED** [review]

**Comment by nitish-awasthi** [comment]
Thanks for working on this @Zenix27 .
Please resolve the conflict by removing these lines.
Conflict

**Comment by agniutkarsh** [comment]
Sure, made the changes, please have a look @nitish-awasthi

File: README.md [hunk 1]
```diff
@@ -2,11 +2,13 @@
 
 Here are some ideas to get you started:
 
+
```
Comment (nitish-awasthi) [inline README.md]: Please remove this extra line! @Zenix27

**Review by agniutkarsh — COMMENTED** [review]
Done

**Review by nitish-awasthi — APPROVED** [review]

**Comment by nitish-awasthi** [comment]
LGTM @Zenix27 
download

**Pull request merged**